src/main/java/nextstep/courses/domain/Session.java

```diff
@@ -5,35 +5,43 @@
 import java.time.LocalDateTime;
 
 public class Session {
-    private final Long id;
+    private Long id;
     private final Long courseId;
+    private Long instructorId;
     private final SessionDuration sessionDuration;
-    private SessionEnrolment sessionEnrolment;
-    private final CoverImage coverImage;
+    private final SessionEnrolment sessionEnrolment;
+    private final CoverImages coverImages;
 
-    public Session(Long id, Long courseId, SessionDuration sessionDuration, SessionEnrolment sessionEnrolment) {
-        this(id, courseId, sessionDuration, sessionEnrolment, null);
-    }
-
-    public Session(Long courseId, SessionDuration sessionDuration, SessionEnrolment sessionEnrolment, CoverImage coverImage) {
-        this(0L, courseId, sessionDuration, sessionEnrolment, coverImage);
+    public Session(Long courseId, SessionDuration sessionDuration, SessionEnrolment sessionEnrolment, CoverImages coverImages) {
+        this.courseId = courseId;
+        this.sessionDuration = sessionDuration;
+        this.sessionEnrolment = sessionEnrolment;
+        this.coverImages = coverImages;
     }
 
-    public Session(Long id, Long courseId, SessionDuration sessionDuration, SessionEnrolment sessionEnrolment, CoverImage coverImage) {
+    public Session(Long id, Long courseId, SessionDuration sessionDuration, SessionEnrolment sessionEnrolment, CoverImages coverImages) {
         this.id = id;
         this.courseId = courseId;
         this.sessionDuration = sessionDuration;
         this.sessionEnrolment = sessionEnrolment;
-        this.coverImage = coverImage;
+        this.coverImages = coverImages;
+    }
+
+    public Apply enrolment(NsUser student, Long userPayed) {
+        sessionEnrolment.enrolment(userPayed);
+        return Apply.defaultApply(this, student);
```

Comment:
```suggestion
        return sessionEnrolment.enrolment(userPayed);
```

sessionEnrolment의 검증이 통과되어야 가능한거라면 sessionEnrolment안에 넣는게 어떨까요?

Reply:
이부분은 아래의 Apply 역할에 대해 질문주신 내용과 함께 수정했습니다!